packages/suite/src/utils/suite/__fixtures__/stake.ts

```diff
@@ -46,3 +46,659 @@ export const transformTxFixtures = [
         },
     },
 ];
+
+export const stakeFixture = [
+    {
+        description: 'should stake 0.1 eth, expect correct transaction object with correct values',
+        args: {
+            from: '0xfB0bc552ab5Fa1971E8530852753c957e29eEEFC',
+            amount: '0.1', // eth
+            symbol: 'eth',
+            identity: '0',
+        },
+        estimatedFee: {
+            success: true,
+            payload: {
+                levels: [{ feeLimit: '21000' }],
+            },
+        },
+        result: {
+            data: '0x3a29dbae0000000000000000000000000000000000000000000000000000000000000001',
+            from: '0xfB0bc552ab5Fa1971E8530852753c957e29eEEFC',
+            gasLimit: 241000, // 21000 + 220000 (reserve)
+            to: '0xD523794C879D9eC028960a231F866758e405bE34',
+            value: '100000000000000000', // wei
+        },
+    },
+    {
+        description: 'should return correct gasLimit when fee estimation is 0',
+        args: {
+            from: '0xfB0bc552ab5Fa1971E8530852753c957e29eEEFC',
+            amount: '0.1', // eth
+            symbol: 'eth',
+            identity: '0',
+        },
+        estimatedFee: {
+            success: true,
+            payload: {
+                levels: [{ feeLimit: '0' }],
+            },
+        },
+        result: {
+            data: '0x3a29dbae0000000000000000000000000000000000000000000000000000000000000001',
+            from: '0xfB0bc552ab5Fa1971E8530852753c957e29eEEFC',
+            gasLimit: 220000, // GAS_RESERVE = 220000
+            to: '0xD523794C879D9eC028960a231F866758e405bE34',
+            value: '100000000000000000', // wei
+        },
+    },
+];
+
+export const stakeFailedFixture = [
+    {
+        description: 'should throw an error when amount is less than minimum',
+        args: {
+            from: '0xfB0bc552ab5Fa1971E8530852753c957e29eEEFC',
+            amount: '0.000000000000000001', // less than minimum
+            symbol: 'eth',
+            identity: '0',
+        },
+        estimatedFee: {
+            success: true,
+            payload: {
+                levels: [{ feeLimit: '21000' }],
+            },
+        },
+        result: 'Min Amount 100000000000000000 wei',
```

Comment:
Do we show this error to a user? 😨

Reply:
I'm not sure about the user but we definitely show it in the browser console. I assume the user can't see these errors because of the field validation.